src/App.vue

```diff
@@ -812,7 +811,7 @@ export default {
         let title = this.getChartTitlePlaceFragment();
 
         this.thresholdChartLayout = {
-          title: `<b>${title}, 1850-2017</b>`,
+          title: `<b>${title}, 1850-2022</b>`,
```

Comment:
Why is this one 2022 instead of 2021 like everywhere else?

Reply:
That's an oversight from when I originally was going to include May 2022 but figured we'll just do all of 2022 in 2022. I've changed this in the most recent commit.